### PR TITLE
🍒 Add 22.4.1 semi structured models metadata

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

**Comment by irinel-birsan-uipath** [comment]
Will be merged at release time

File: metadata/W9__3__metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "mlPackageLanguage": "PYTHON37_DU",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Document Understanding",
+    "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "22.4",
+    "languageVersion": 4,
+    "version": 3,
+	"customVersion": "22.4.1",
```
Comment (vnaren23) [inline metadata/W9__3__metadata.json:23]: nit: Indentation incorrect across all files

**Review by vnaren23 — APPROVED** [review]
LGTM

**Review by florin-szilagyi — APPROVED** [review]

File: metadata/W9__3__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/W9__3__metadata.json:23]: locally looks fine

**Pull request merged**